content/applications/inventory_and_mrp/inventory/management/products/strategies.rst

```diff
@@ -1,75 +1,202 @@
+  once the :guilabel:`On Hand` quantity goes below the :guilabel:`Min Quantity`) or
+  :guilabel:`Manual` (only when the replenishment is requested).
+- :guilabel:`Procurement Group`: the reference number for how the product is being acquired, such
+  as a sales order, purchase order, or manufacturing order.
+- :guilabel:`Min Quantity`: the minimum amount of product that should be available. When inventory
+  levels goes below this number, the replenishment is triggered.
+- :guilabel:`Max Quantity`: the amount of product that should be available after replenishing the
+  product.
+- :guilabel:`Multiple Quantity`: if the product should be ordered in specific quantities, enter the
+  number that should be ordered. For example, if the :guilabel:`Multiple Quantity` is set to `5`,
+  and only 3 are needed, 5 products are replenished.
+- :guilabel:`To Order`: the amount of product that is currently needed, and will be ordered, if the
+  :guilabel:`Order Once` or :guilabel:`Automate Orders` button is clicked.
+- :guilabel:`UoM`: the unit of measure used to acquire the product.
+- :guilabel:`Company`: the company for which the product is acquired.
+
+By default, the quantity in the :guilabel:`To Order` field is the quantity required to reach the set
+:guilabel:`Max Quantity`. However, the :guilabel:`To Order` quantity can be adjusted by clicking on
+the field and, changing the value, and clicking :guilabel:`Save`. To replenish a product manually,
+click :guilabel:`Order Once`.
+
+To automate a replenishment from the :guilabel:`Replenishment` page, click :guilabel:`Automate
+Orders` on the right-side of the line, represented by a :guilabel:`🔄 (circular arrow)` icon.
+
+When this button is clicked, Odoo will automatically generate a draft :abbr:`PO (Purchase Order)`/
+:abbr:`MO (Manufacturing Order)` every time the forecasted stock level falls below the set
+:guilabel:`Min Quantity` of the reordering rule.
+
+On the :guilabel:`Replenishment` page, a reordering rule or manual replenishment can be temporarily
+deactivated for a given period, by clicking the :guilabel:`🔕 (snooze)` icon on the far-right of the
+line.
+
+.. image:: strategies/reordering-rule-snooze-settings.png
+   :align: center
+   :alt: Snooze options to turn off notifications for reordering for a period of time.
 
-Then, click on your product to access the related product form and, on
-the "Inventory submenu", do not forget to select a supplier.
+A :abbr:`PO (Purchase Order)` or :abbr:`MO (Manufacturing Order)` created by a manual replenishment
+has a :guilabel:`Replenishment Report` as the source document. A :abbr:`PO (Purchase Order)` or
+:abbr:`MO (Manufacturing Order)` created by an automated reordering rule has the :abbr:`SO (sales
+order(s))` reference number(s) that triggered the rule as the source document.
 
-.. image:: strategies/strategies02.png
+.. image:: strategies/rfq-source-document.png
```

Comment:
This image is too wide at 1440px, please retake to be 933px wide. It is not necessary to show the entire screen, you could probably get away with showing just the Source Document, Total, Status and additional icon button with the 3 rows below